src/lib/migrations/routines/01-update-search-assistants.ts

```diff
@@ -0,0 +1,49 @@
+import type { Migration } from ".";
+import { getCollections } from "$lib/server/database";
+import type { AnyBulkWriteOperation } from "mongodb";
+import type { Assistant } from "$lib/types/Assistant";
+
+const migration: Migration = {
+	guid: "78c019ad-6bac-4958-a8ad-57a880e79bbf",
```

Comment:
Any reason for not using `ObjectId`? (which could be the `_id` of the `migrationResults` collection)

Reply:
Good catch, fixed it!